kratos/sources/constitutive_law.cpp

```diff
@@ -548,13 +548,21 @@ bool ConstitutiveLaw::IsIncremental()
  * @param rMaterialProperties the Properties instance of the current element
  * @param rElementGeometry the geometry of the current element
  * @param rShapeFunctionsValues the shape functions values in the current integration point
+ * @param rCurrentProcessInfo the current process information
  */
 void ConstitutiveLaw::InitializeMaterial(const Properties& rMaterialProperties,
         const GeometryType& rElementGeometry,
         const Vector& rShapeFunctionsValues)
 {
 }
 
+void ConstitutiveLaw::InitializeMaterial(const Properties& rMaterialProperties,

```

Comment:
LGTM!

Please add a deprecation warning to the other function (KRATOS_DEPRECATED_MESSAGE)